cluster/config-defaults.yaml

```diff
@@ -236,3 +236,6 @@ custom_resource_webhook_conversion: "false"
 node_max_pods: "110" # Default: 110
 # flag passed to controller-manager
 node_cidr_mask_size: "24" # Default: 24
+
+# when set to true, routes external traffic to the apiserver through a skipper sidecar
+apiserver_proxy: "false"
```

Comment:
I would suggest enabling this by default. We can start by opting out in all clusters by adding `apiserver_proxy: "false"` to all existing clusters.

Reply:
Introducing this at the same time as enabling it won't work and e2e will fail. I suggest we keep it like that and switch the default in a separate PR. This way we can use e2e to test the update paths as well.